PocketKit/Sources/PocketKit/Article/ReadableViewModel/RecommendationViewModel.swift

```diff
@@ -195,7 +224,7 @@ extension RecommendationViewModel {
         track(identifier: .itemFavorite)
     }
 
-    private func unfavorite() {
+    internal func unfavorite() {
```

Comment:
```suggestion
    func unfavorite() {
```

```suggestion
    func favorite() {
```

Internal by default; no need to explicitly mark it as such.